### ISSUE TITLE
how do I sweep a paper wallet with CoPay v3.4.0? 

### ISSUE THREAD

**Issue by salty1965** [bitpay/copay#6208]
I was watching this guys video https://youtu.be/VTsHeiBhPIM on how to Send (which I accomplished) to a paper wallet then sweep the paper wallet (which I have not been able to.) Because clearly the versions seem different. his buttons are not the same as my CoPay Version 3.4.0

do have an Android Samsung-SM-G920A cell phone 

please help thank you.

**Comment by gabrielbazan7** [comment]
Just scan the private key you will get an option to sweep paper wallets

**Comment by RussianSwiss** [comment]
Touch "Send" on a bar and scan privkey QR with camera.

**Comment by JDonadio** [comment]
https://github.com/bitpay/copay/issues/5806#issuecomment-288500830

**Comment by salty1965** [comment]
thank you every one i was scanning the address. i am new. thank you some much. ROCK ON

**Comment by JDonadio** [comment]
Glad to help!

**Comment by jasmreid** [comment]
It seems the latest version of the Copay wallet does not support "sweep". None of the menu items discussed above work the same in the current version (3.9.1) on Windows. When I hit Send the only options are Buy Bitcoin and Show Bitcoin address. I created a new BCH Wallet in the Copay application to receive the BCH coin I have after the hard fork. I backed it up. Nowhere is there an opportunity to enter a private key to get the wallet to sweep the BCH chain to find funds. Any clues??
